internal/monitors/traefik/metadata.yaml

```diff
@@ -0,0 +1,297 @@
+monitors:
+- monitorType: traefik
+  doc: |
+    ### DESCRIPTION
+
+    Configure <a target="_blank" href="https://docs.traefik.io/configuration/metrics">Traefik</a> to expose internal
+    metrics as prometheus metrics at an endpoint. Install the
+    <a target="_blank" href="https://github.com/signalfx/signalfx-agent">SignalFx Smart Agent</a>
+    and configure it to access SignalFx and use the
+    <a target="_blank" href="https://github.com/signalfx/signalfx-agent/blob/9feb3f77fdf6de46dc476f62568ad4f9b725660c/docs/monitors/prometheus-exporter.md">prometheus-exporter</a> monitor.
+
+    ### REQUIREMENTS AND DEPENDENCIES
+
+    | Software          | Version        |
+    |-------------------|----------------|
+    | signalfx-agent    |     3.4+       |
+
+    ### INSTALLATION
+
+    Follow these steps to install this plugin:
```

Comment:
Get rid of these sections, just make a section describing how to configure Traefik and call it something like "# Configure Traefik".

Also, don't use the word "plugin" anywhere here unless it is some Traefik terminology.